docs/store/README.md

```diff
@@ -20,6 +20,7 @@ Users can use the following software solutions to store data on the Filecoin net
 
 The following applications allow you to store data on the Filecoin network with no technical setup:
 
+- [Web3.Storage](https://web3.storage) is an IPFS pinning service and Filecoin storage platform built-in-one. Upload files to Web3.Storage and access them instantly, safe in the knowledge that your data is securely backed up using the Filecoin network.
```

Comment:
This list should probably be sorted alphabetically, its getting a bit crowded. A sorted list makes it easier for users to find things.